pkg/berglas/runtime.go

```diff
@@ -205,3 +210,58 @@ type cloudRunContainer struct {
 		Value string `json:"value"`
 	} `json:"env"`
 }
+
+// gaeEnv is a Google App Engine environment.
+type gaeEnv struct{}
+
+type appengineVersion struct {
+	EnvVariables map[string]string `json:"envVariables"`
+}
+
+// EnvVars returns the list of envvars set on this app engine version
+func (e *gaeEnv) EnvVars(ctx context.Context) (map[string]string, error) {
```

Comment:
Does this work the same on flex and standard?

Reply:
I believe it does - however I haven't been able to test that e2e yet. I'm just finishing up my Python library so I'll try it with that and update here if required.